backend/EasyMeets.Core/EasyMeets.Core.WebAPI/Validators/Sample/NewSampleDtoValidator.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using EasyMeets.Core.Common.DTO;
 using FluentValidation;
 
-namespace EasyMeets.Core.WebAPI.Validators
+namespace EasyMeets.Core.WebAPI.Validators.Sample
```

Comment:
Why did you add this?

Reply:
It would be better to have one namespace for all validators instead to have a separate namespace for everyone

Reply:
Okay, I will fix it soon

Reply:
Fixed